web_external/js/views/body/ImagesView/HistogramPane.js

```diff
@@ -71,6 +71,8 @@ isic.views.ImagesViewSubViews.HistogramPane = isic.View.extend({
             });
         attributeSections.select('.content').each(_.bind(function (d) {
             var histogramId = isic.shims.makeValidId(d + '_histogramContent');
+            // Destroy all tooltips before re-rendering, as a precaution
+            $('.tooltip').remove();
```

Comment:
If you wanted to leave the image tooltips alone, you could just class them differently and scope this selector to just the histogram tooltips right?